### PR TITLE
ocaml-planet.0.1 - via opam-publish

### PR DESCRIPTION
**Pull request by kayceesrk**
A planet (feed aggregator) in OCaml.

A library for aggregating RSS2 and Atom feeds in OCaml.

Features:

* Performs deduplication.
* Sorts the posts from most recent to oldest.
* Supports pagination and generating well-formed html prefix snippets.
* Depends on ocamlnet for html parsing.

---
* Homepage: https://github.com/kayceesrk/ocaml-planet
* Source repo: https://github.com/kayceesrk/ocaml-planet.git
* Bug tracker: https://github.com/kayceesrk/ocaml-planet/issues

---
Pull-request generated by opam-publish v0.2.1

### CODE REVIEW

File: packages/ocaml-planet/ocaml-planet.0.1/opam [hunk 1]
```diff
@@ -0,0 +1,23 @@
+opam-version: "1.2"
+maintainer: "sk826@cl.cam.ac.uk"
+authors: "KC Sivaramakrishnan"
+homepage: "https://github.com/kayceesrk/ocaml-planet"
+bug-reports: "https://github.com/kayceesrk/ocaml-planet/issues"
+license: "ISC"
+dev-repo: "https://github.com/kayceesrk/ocaml-planet/irmin.git"
```
Comment (samoht) [inline packages/ocaml-planet/ocaml-planet.0.1/opam]: there's a typo here I think :p

Comment (kayceesrk) [inline packages/ocaml-planet/ocaml-planet.0.1/opam]: Yes. :-) Thanks. Fixed it. There is a missing dependency (lwt) in the opam file, causing builds to fail. I'll resubmit the pull request.

Comment (avsm) [inline packages/ocaml-planet/ocaml-planet.0.1/opam]: No need to resubmit.  You can just add changesets to your branch and they'll get built.  When it's all green, you can rebase the branch if desired.

**Comment by avsm** [comment]
Nice library!

**Pull request merged**